hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -151,41 +156,71 @@ class HassioIngressView extends LitElement {
       addon = await fetchHassioAddonInfo(this.hass, addonSlug);
     } catch (err: any) {
       await showAlertDialog(this, {
-        text: "Unable to fetch add-on info to start Ingress",
+        text: this.supervisor.localize("ingress.error_addon_info"),
         title: "Supervisor",
       });
       await nextRender();
-      history.back();
+      navigate("/hassio/store", { replace: true });
       return;
     }
 
-    if (!addon.ingress_url) {
+    if (!addon.version) {
       await showAlertDialog(this, {
-        text: "Add-on does not support Ingress",
+        text: this.supervisor.localize("ingress.error_addon_not_installed"),
         title: addon.name,
       });
       await nextRender();
-      history.back();
+      navigate(`/hassio/addon/${addon.slug}/info`, { replace: true });
       return;
     }
 
-    if (addon.state !== "started") {
+    if (!addon.ingress_url) {
       await showAlertDialog(this, {
-        text: "Add-on is not running. Please start it first",
+        text: this.supervisor.localize("ingress.error_addon_not_supported"),
         title: addon.name,
       });
       await nextRender();
-      navigate(`/hassio/addon/${addon.slug}/info`, { replace: true });
+      history.back();
       return;
     }
 
+    if (addon.state !== "started") {
+      const confirm = await showConfirmationDialog(this, {
```

Comment:
Why?
Last time we talked about this, we did not want to throw dialogs against the user where it was not needed?
If a user click the entry of an add-on, they do that to get to it.
Adding a dialog makes it another action they need to do, and it will add to the. total time from intent to restult

Reply:
Sure, something is better, but it's still an unessesarry stop.

Reply:
🤷‍♂️ depends, if I by accident click on a link should it start that addon without confirmation?